src/region.c

```diff
@@ -101,6 +101,29 @@ gdip_is_Point_in_RectFs_Visible (float x, float y, GpRectF* r, int cnt)
 	return FALSE;
 }
 
+static BOOL
+gdip_is_Rect_in_RectF_Visible (float x, float y, float width, float height, GpRectF* rect)
+{
+	if (rect->Width == 0 || rect->Height == 0)
+		return FALSE;
+
+	return x < rect->X + rect->Width && x + width > rect->X && y < rect->Y + rect->Height && y + height > rect->Y;
```

Comment:
`x + width > rect->X` -> `x + width >= rect->X`
`y + height > rect->Y` -> `y + height >= rect->Y`

Reply:
Doing this change causes the tests to fail. It seems that the bounds are not inclusive on the right and bottom